lib/rubocop/cop/style/negated_while.rb

```diff
@@ -4,6 +4,31 @@ module RuboCop
   module Cop
     module Style
       # Checks for uses of while with a negated condition.
+      #
+      # @example
+      #   # bad
+      #
```

Comment:
Can you remove the blank lines between the comment labels and the code?

Reply:
@bbatsov sorry, fixed